tests/bdd/scale/test_NAS_T1124.py

```diff
@@ -0,0 +1,111 @@
+    assert wait_on_element(driver, 10, '//span[contains(.,"Dashboard")]')
+    assert wait_on_element(driver, 10, '//span[contains(text(),"System Information")]')
+    assert wait_on_element(driver, 5, '//mat-list-item[@ix-auto="option__Shares"]', 'clickable')
+    driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Shares"]').click()
+
+
+@then('The Windows Shares(SMB) page should open, Click Add')
+def the_windows_sharessmb_page_should_open_click_add(driver):
+    """The Windows Shares(SMB) page should open, Click Add."""
+    assert wait_on_element(driver, 5, '//h1[contains(text(),"Sharing")]')
+    assert wait_on_element(driver, 5, '//mat-card[contains(.,"SMB")]//button[@ix-auto="button__-add"]', 'clickable')
+    driver.find_element_by_xpath('//mat-card[contains(.,"SMB")]//button[@ix-auto="button__-add"]').click()    
+
+
+@then(parsers.parse('Set Path to the ACL dataset "{path}", Input "{smbname}" as name, Click to enable, Input "{description}" as description, and Click Summit'))
+def set_path_to_the_acl_dataset_mnttanktank_acl_dataset_input_mytankshare_as_name_click_to_enable_input_my_tank_test_share_as_description_and_click_summit(driver, path, smbname, description):
+    """Set Path to the ACL dataset "{path}", Input "{smbname}" as name, Click to enable, Input "{description}" as description, and Click Summit."""
+    time.sleep(1)
+    global smb_path
+    smb_path = path
+    """Set Path to the ACL dataset "/mnt/system/tank_acl_dataset"."""
+    assert wait_on_element(driver, 5, '//input[@ix-auto="input__path"]', 'inputable')
+    driver.find_element_by_xpath('//input[@ix-auto="input__path"]').clear()
+    driver.find_element_by_xpath('//input[@ix-auto="input__path"]').send_keys(path)
+    assert wait_on_element(driver, 5, '//input[@ix-auto="input__Name"]', 'inputable')
+    driver.find_element_by_xpath('//input[@ix-auto="input__Name"]').clear()
+    driver.find_element_by_xpath('//input[@ix-auto="input__Name"]').send_keys(smbname)
+    assert wait_on_element(driver, 5, '//mat-checkbox[@ix-auto="checkbox__Enabled"]', 'clickable')
+    checkbox_checked = attribute_value_exist(driver, '//mat-checkbox[@ix-auto="checkbox__Enabled"]', 'class', 'mat-checkbox-checked')
+    if not checkbox_checked:
+        driver.find_element_by_xpath('//mat-checkbox[@ix-auto="checkbox__Enabled"]').click()
+    assert attribute_value_exist(driver, '//mat-checkbox[@ix-auto="checkbox__Enabled"]', 'class', 'mat-checkbox-checked')
+    time.sleep(1)
+    assert wait_on_element(driver, 5, '//input[@ix-auto="input__Description"]', 'inputable')
+    driver.find_element_by_xpath('//input[@ix-auto="input__Description"]').clear()
+    driver.find_element_by_xpath('//input[@ix-auto="input__Description"]').send_keys(description)
+    assert wait_on_element(driver, 5, '//button[@ix-auto="button__SAVE"]', 'clickable')
+    driver.find_element_by_xpath('//button[@ix-auto="button__SAVE"]').click()
+
```

Comment:
You should use assert wait_on_element_disappear(driver, 15, '//h6[contains(.,"Please wait")]') to make sure that the job is completed.